### PR TITLE
Added team names and ids

### PR DESCRIPTION
**Pull request by MostafaOmar98**

### CODE REVIEW

**Review by ahmeddrawy — APPROVED** [review]

File: labs/task 1/Main.java [hunk 1]
```diff
@@ -1,3 +1,6 @@
+// Mostafa Omar Mahmoud 20170292 CS_IS_2
+// Ahmed Mohamed Hanafy 20170377 CS_IS_2
```
Comment (ahmeddrawy) [inline labs/task 1/Main.java:2]: 20170357

**Comment by ahmeddrawy** [comment]
I reverted because the my ID was wrong

**Comment by MostafaOmar98** [comment]
احا انا بعتلها الميل فيه 20170377

**Comment by MostafaOmar98** [comment]
بعتلها ايميل بالتعديل ياسطى اسف

**Pull request merged**